.github/CODEOWNERS

```diff
@@ -3,4 +3,4 @@
 # Each line is a file pattern followed by one or more owners.
 
 # Harmony owns any files in the .github directory at the root of the repository and any of its subdirectories.
```

Comment:
Should we update the comment too as it mentions `Harmony`?

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/pull/8995/commits/0bfc7052cdb78073ecdf738a0104cb4393084aae